api/src/services/authorization.js

```diff
@@ -181,10 +226,6 @@ const filterAllowedDocs = (authorizationContext, docObjs) => {
   return allowedDocs;
 };
 
-const alwaysAllowCreate = doc => {
```

Comment:
question: This was no longer needed because `feedback` docs are stored in `medic-users-meta` correct?  Are we concerned about an upgrade scenario where clients have existing `feedback` docs offline in their `medic` db (from pre-upgrade) and now, post-upgrade, these docs will not sync?

Reply:
We moved feedback docs from medic to users-meta back in `3.5.0` (https://github.com/medic/cht-core/issues/4898) . I am not concerned about projects upgrading from 3.5.0 anymore.

Reply:
:+1:  Perfect!